teal/crates/init.tl

```diff
@@ -43,6 +43,8 @@ local record M
     -- Open the `crates.io` page of the crate on the current line.
     open_crates_io: function()
 
+    -- Returns a boolean indicating whether there is information to show in a popup.
+    popup_available: function()
```

Comment:
We should probably add the return type here and update the `gen_doc.lua` script to accept that. I can do that later this weekend when I have time.